src/unum/log/log_common.h

```diff
@@ -21,6 +21,19 @@
 // Max log file pathsname length the code will deal with
 #define LOG_MAX_PATH 128
 
+// A file containing the Prefix(directory)
+// All log files (For example unum.log)
+// will be prefixed with the content in this filename
+#ifdef PERSISTENT_FS_DIR_PATH
+#define LOGS_PREFIX_FILE PERSISTENT_FS_DIR_PATH"/unum_prefix_file.txt"
+#else
+#define LOGS_PREFIX_FILE "/etc/" // Default
+#endif
+#define UNUM_LOG_CONSOLE_NAME        "/dev/console"
```

Comment:
Please separate 
```
#define UNUM_LOG_SCALE_FACTOR 4
#define UNUM_LOG_EXTRA_ROTATIONS 2
#define UNUM_LOG_CUT_FRACTION 10
```
from what's above and 
and add detailed explanation what each constant is responsible for (preferably with an example),
and note how the platform can override if needed (i.e. undef and redefine in platform's log.h...)

Reply:
Explained in detail.